jwst/cube_build/cube_build_wcs_util.py

```diff
@@ -108,6 +111,22 @@ def find_corners_MIRI(input, this_channel, instrument_info, coord_system):
     lambda_min = np.nanmin(lam)
     lambda_max = np.nanmax(lam)
 
+    # before returning ra should be between 0 to 360
```

Comment:
I know this is just a code comment and not part of a novel, but without suitable punctuation in this sentence it at first reads (at least to me) "before returning ra ... something happens" (i.e. do some action before returning ra), whereas what you want to say is "before returning, ra should be ..."